charts/cloudserver/values.yaml

```diff
@@ -49,6 +49,7 @@ proxy:
   # Additionally you can pass a CA certifcate that will be added to the trusted
   # certs. If the proxy URL is configured and caCert is true, helm
   # will look for a file named 'ca.crt' at the root path of this chart.
+  # Note: To avoid unexpected behavior, only specify one of "http" or "https" proxy options.
```

Comment:
minor but lets-do-this-while-its-hot: can you wrap the line so that it fits into 80 columns?

Reply:
Done!